tests/unittests/Foundation/NSIndexSetTests.mm

```diff
@@ -27,18 +27,20 @@ void verifyIndexSetEqualsArray(NSArray* expected, NSIndexSet* actual) {
     }];
 
     ASSERT_EQ([expected count], callCount);
-    ASSERT_EQ(YES, [actual isEqualToIndexSet:expectedIndexSet]);
+    ASSERT_TRUE([actual isEqualToIndexSet:expectedIndexSet]);
```

Comment:
That worries me, because isEqual: should definitely call through to isEqualToIndexSet: